docs/content/manual/rancher-integration/drain-using-rancher-ui.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Drain using Rancher UI
+---
+
+Test with Longhorn default setting of 'Node Drain Policy': `block-if-contains-last-replica`
+----
+
+### 1. Drain operation on single node using Rancher UI
+
+**Given**  Single node cluster with Longhorn installed
+
+AND few RWO and RWX volumes attached with node/pod exists
+
+AND 1 RWO and 1 RWX volumes unattached
+
+**When** Drain the node with default values of Rancher UI
+
+**Then** Drain should be blocked
+
+AND Volumes should be healthy and data should be intact after drain completes
+
+AND The volumes which were attached initially should be attached and the ones which is unattached should be unattached
+
+AND Logs should not have any unexpected errors
+
+
+### 2. Drain operation on a multiple node using Rancher UI - default value
+
+**Given**  Cluster with 1 etcd/control plane and 3 worker nodes with Longhorn installed
+
+AND few RWO and RWX volumes attached with node/pod exists
+
+AND few RWO and RWX volumes unattached
+
+**When** Drain the worker node one by one with default values of Rancher UI
+
+**Then** Drain should be successful
+ 
+AND Volumes should be healthy and data should be intact after drain completes
+
+AND The volumes which were attached initially should be attached and the ones which is unattached should be unattached
+
+AND Logs should not have any unexpected errors
+
+
+### 3. Drain operation on a multiple node using Rancher UI - Force
+
+**Given**  Cluster with 1 etcd/control plane and 3 worker nodes with Longhorn installed
+
+AND few RWO and RWX volumes attached with node/pod exists
+
+AND few RWO and RWX volumes unattached
+
+**When** Drain the worker node one by one with Force option enabled using Rancher UI.
+
+**Then** Drain should be successful
+
+AND Volumes should be healthy and data should be intact after drain completes
+
+AND The volumes which were attached initially should be attached and the ones which is unattached should be unattached
+
+AND Logs should not have any unexpected errors.
+
+
+### 4. Drain operation on a multiple node using Rancher UI - Single replica of volumes
+
+**Given**  Cluster with 1 etcd/control plane and 3 worker nodes with Longhorn installed
+
+AND few RWO and RWX volumes with 1 replica attached with node/pod exists
+
+AND few RWO and RWX volumes with 1 replica unattached
+
+**When** Drain the worker node one by one, start with the node which has volume's replica
+
+**Then** Drain should be successful
+
+AND Volumes should be healthy and data should be intact after drain completes
+
+AND The volumes which were attached initially should be attached and the ones which is unattached should be unattached
+
+AND Logs should not have any unexpected errors
+
```

Comment:
In this case, I enabled both `Delete Empty Dir Data` and `Force` when drain from Rancher UI, I can see volume move from drain node to another node, but replica still running on drain node and from Rancher UI the node state was `cordoned` not `drained`.

In addition, if i change Node Drain Policy to `always allow`, the drain success.